conformance/Makefile

```diff
@@ -3,7 +3,7 @@ PREFIX = nginx-gateway-fabric
 NGINX_PREFIX = $(PREFIX)/nginx
 GW_API_VERSION ?= 1.0.0
 GATEWAY_CLASS = nginx
-SUPPORTED_FEATURES = HTTPRoute,HTTPRouteQueryParamMatching,HTTPRouteMethodMatching,HTTPRoutePortRedirect,HTTPRouteSchemeRedirect,GatewayClassObservedGenerationBump
```

Comment:
Why are `HTTPRoute` and `GatewayClassObservedGenerationBump` removed?

Reply:
HTTPRoute is enabled by default, and GatewayClassObservedGenerationBump is covered by GatewayPort8080